content/ADR-175-remove-assets-from-build-local-addressables.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: adr
+adr: 175
+title: Remove assets from Renderer's build. Addressables Integration
+date: 2023-01-16
+status: Review
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+  - ajimenezDCL
+  - mikhail-dcl
+---
+
+## Abstract
+This document describes an approach on how to make the reference's client lightweight by streaming the currently embedded assets. 
+
+## Problem
+Every nice UI, sound effect, particle or material we add to the renderer increases its size. This directly impact loading times or caching since the client's size will go above the max allowed for it. The idea is to pack textures, audio files and other assets, and stream them in runtime. 
+
+## Solution
+### Pack the Assets
+Similar to what we are doing with scene assets, we can pack everything into one or multiple (to always be under the cache size limit) asset packs to be consumed by the client. This solution should be part of a different system than the one used for scene asset.
```

Comment:
it would be nice to have at least small remark why we can't re-use scene asset approach and these solutions should be different

Reply:
I think it's about completely different systems: one is served for user-generated content and another is entirely under our control.
I have doubts if and what I should write here. Considering it's a piece from @AjimenezDCL , maybe you have any ideas?

Reply:
>other stuff
Can you elaborate more? I will put it in the ADR

Reply:
Like the library which handles a ref-counted collection to dispose and release loaded items, built-in tools to read the metadata and metrics files and it's based in promises, which we hate hahaha.

Reply:
Addressed